recipes/toytree/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "toytree" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/eaton-lab/toytree/archive/{{ version }}.tar.gz
+  sha256: 5c3e5262060bdb1c4fb16c5031a554b285bcda9b243a6cd2e5bef8b151c23252
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```

Comment:
There are a few version restriction discrepancies with the `setup.py`:

```
install_requires=[
        "toyplot>=0.18.0",
        "numpy>=1.8.0",
        "future",
        "requests",
        "six",
    ],
```

I'm not sure how important the version differences are. Also leaving off `six` is ok because we're only building for py3.

Reply:
Thanks for checking the recipe.

Is building only for py3 a conda-forge restriction?

I can remove the version restrictions.

Reply:
I just pushed a new tag with setup.py modified to remove six.

Reply:
Just to be clear about a few things, conda-forge allows feedstocks to add back in python 2 support if needed. From https://conda-forge.org/docs/user/announcements.html#announcements : 

> 2020-03-21
> Python 2.7 Admin Command Available
> 
> A webservices admin command is now available to add Python 2.7 back to feedstocks. Put `@conda-forge-admin add python 2.7` in the title on an issue in your feedstock. The admin webservices bot will then issue a PR adding back Python 2.7. Note that this PR will remove other Python builds and any win, aarch64, or ppc64le builds. If you want to keep those, merge the PR into a separate branch on your feedstock.

I just want to make sure this is clear because looking upstream it appears that you dropped python2 support with the changes you made in `1.1.15`, but the meta info in the `setup.py` says that python2 is still supported. You don't have to drop python 2 support in your library to be on conda-forge. Conda-forge just won't build python 2 binaries by default, even if the library still supports python 2 (for users who want to download source code from github as an example).

Reply:
six wasn't really required for py2 compatibility, I was just using it out of laziness. The changes I made are py2/3 compatible (passing tests and I also just tested installing 1.1.15 from my conda channel in a fresh docker py27 miniconda env). If I understand you right then I don't think we need to add special py27 feedstock, since this is noarch build anyways. 

Are there any other changes required to add this to conda-forge?